tests/screenshot/run_differ.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python2.7
+# Compresses the core Blockly files into a single JavaScript file.
+#
+# Copyright 2019 Google Inc.
+# https://developers.google.com/blockly/
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+# run_differ.py with no parameters runs all screenshot tests with blocks in rtl
+# and not collapsed.
+# --name <test_name> runs all tests that contain the given name.
+# --collapsed runs all tests with the blocks collapsed
+# --insertionMarker runs all tests with the blocks set as insertion markers.
+#
+
+import os, errno, platform, shutil, sys
+
+NAME_ARG = "--name"
+COLLAPSE_ARG = "--collapsed"
+RTL_ARG = "--rtl"
+INSERTION_ARG = "--insertionMarker"
+
+def main():
+  cleanup()
+  filter_text = find_filter_text()
+  is_rtl = change_to_rtl()
+  collapse = should_collapse()
+  isInsertionMarker = check_insertion_marker()
+  gen_screenshots(filter_text, collapse, isInsertionMarker)
+  diff_screenshots(filter_text)
+  display_screenshots()
+  cleanup_rtl(is_rtl)
+
+def cleanup():
+  remove_dir("tests/screenshot/outputs/new")
+  remove_dir("tests/screenshot/outputs/diff")
+  remove_file("tests/screenshot/outputs/test_output.js")
+  remove_file("tests/screenshot/outputs/test_output.json")
+
+def find_filter_text():
+  args = sys.argv
+  for i in range(len(args)):
+    if args[i] == NAME_ARG:
+      if i + 1 < len(args):
+        return args[i+1]
+      else:
+        print("Must supply a name after name arg")
+        sys.exit()
+  return ""
+
+def change_to_rtl():
+  args = sys.argv
+  if RTL_ARG in args:
+    os.system("sed -i -e 's/rtl: false/rtl: true/g' tests/screenshot/playground_new.html")
```

Comment:
Doesn't this permanently change the file?

Another option is to take the inject call out of the playground_old and playground_new files and have it in gen_screenshots.js.  Then you can make other changes to it (e.g. zoom level) and have them apply to both playgrounds at once.